iwyu.cc

```diff
@@ -898,21 +898,13 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
         const_cast<CXXDestructorDecl*>(dtor), parent_type, expr);
   }
 
-  // This is to catch assigning template functions to function pointers.
+  // This is to catch binding of function pointers to templates.
```

Comment:
I was worried "binding" was too semantically overloaded, and I see what you're saying. I regret naming the whole test case "bind_funcptr" now :-)

I'll evalute your test case in context with the others and see if it makes sense there too.